README.md

```diff
@@ -25,7 +25,7 @@ Most of the TiSpark logic is inside a thin layer, namely, the [tikv-client](http
 - [Getting Started](https://github.com/pingcap/tispark/wiki/Getting-Started)
 - [User Guide](https://github.com/pingcap/tispark/blob/master/docs/userguide.md)
 - [Authorization and authentication](./docs/authorization_userguide.md)
-- [Configuration](https://github.com/pingcap/tispark/wiki/Configuration)
+- [Configuration](./docs/configuration.md)
```

Comment:
No stale read ToC entry, delibrately?

Reply:
stale read's link put in the configuration.md. After all, a TOC per feature may too messy